ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -46,7 +46,7 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		}
 	}
 	var progressSummarizer : ProgressSummarizer?
-	var queryRefreshControl: UIRefreshControl?
+	var refreshController: UIRefreshControl?
```

Comment:
@hosy would prefer `refreshControl` as a name here

Reply:
I don´t understand what you mean. The current naming is `refreshControl`.

Reply:
@hosy it is called `refreshController`, not `control`

Reply:
Ah! Thanks for your hint! Yes, you are right!

Reply:
Just saw, that this it not possible, because UITableViewController has already an property called `refreshControl`: Cannot override with a stored property 'refreshControl'.

Reply:
@hosy Ok, fine. That's an argument:) Will approve.